core/library.go

```diff
@@ -56,24 +56,26 @@ type BuildProps struct {
 	Library_version string
 
 	// The list of shared lib modules that this library depends on.
-	// 'shared_libs' only makes sense for share libraries and executables -
-	// static libs should use 'export_shared_libs', so the dependency is
-	// propagated to the top-level object.
+	// These are propagated to the closest linking object when specified on static libraries.
```

Comment:
Please update docs/module_types/common_module_properties.md for shared_libs, static_libs and ldlibs so that it contains the same text. We can skip updating the documentation on the export properties, as we will delete them shortly